Assets/MRTK/Tests/PlayModeTests/Utils/TestButtonUtilities.cs

```diff
@@ -84,6 +84,11 @@ public enum DefaultButtonType
         /// Instantiate and configure one of the <see cref="DefaultButtonType"/> types.
         /// Returns reference to <see cref="Microsoft.MixedReality.Toolkit.UI.Interactable"/> component and Transform to movable object that transforms on press 
         /// </summary>
+        /// <remarks>
+        /// The button will be instantiated relative to the current MRTKPlayspace. The is, its world transform
```

Comment:
```suggestion
        /// The button will be instantiated relative to the current MRTKPlayspace. This means that its world transform
```
Or something like that?